ios/src/TGMapViewController.mm

```diff
@@ -1,101 +1,97 @@
 //
-//  ViewController.m
+//  TGMapViewController.mm
 //  TangramiOS
 //
 //  Created by Matt Blair on 8/25/14.
-//  Copyright (c) 2014 Mapzen. All rights reserved.
+//  Updated by Matt Smollinger on 7/29/16.
+//  Copyright (c) 2016 Mapzen. All rights reserved.
 //
 
-#import "ViewController.h"
+#import "TGMapViewController.h"
 #include "platform_ios.h"
 
-@interface ViewController () {
+@interface TGMapViewController ()
 
-}
-@property (strong, nonatomic) EAGLContext *context;
-@property CGFloat pixelScale;
-@property bool renderRequested;
-
-- (void)setupGL;
-- (void)tearDownGL;
-- (void)respondToTapGesture:(UITapGestureRecognizer *)tapRecognizer;
-- (void)respondToDoubleTapGesture:(UITapGestureRecognizer *)doubleTapRecognizer;
-- (void)respondToPanGesture:(UIPanGestureRecognizer *)panRecognizer;
-- (void)respondToPinchGesture:(UIPanGestureRecognizer *)pinchRecognizer;
+@property (nullable, strong, nonatomic) EAGLContext *context;
+@property (assign, nonatomic) CGFloat pixelScale;
+@property (assign, nonatomic) BOOL renderRequested;
 
 @end
 
 
-@implementation ViewController
+@implementation TGMapViewController
 
 - (void)viewDidLoad
 {
     [super viewDidLoad];
-
+    
```

Comment:
Trailing spaces should be trimmed, even on empty lines. I think you can configure Xcode to do this for you.